admin/views/tabs/metas/post-types.php

```diff
@@ -27,61 +29,76 @@
 
 $view_utils = new Yoast_View_Utils();
 
+echo '<p>';
+esc_html_e( 'The settings on this page allow you to specify what the default search appearance should be for any type of content you have. You can choose which content types appear in search results and what their default description should be.', 'wordpress-seo' );
+echo '</p>';
+
 if ( is_array( $post_types ) && $post_types !== array() ) {
-	foreach ( $post_types as $post_type ) {
-		echo '<div class="tab-block" id="' . esc_attr( $post_type->name . '-titles-metas' ) . '">';
-		echo '<h2 id="' . esc_attr( $post_type->name ) . '">' . esc_html( $post_type->labels->name ) . ' (<code>' . esc_html( $post_type->name ) . '</code>)</h2>';
+	foreach ( $post_types as $id => $post_type ) {
+		$single_label = $post_type->labels->singular_name;
+		$plural_label = $post_type->labels->name;
+
+		echo '<div class="paper tab-block" id="' . esc_attr( $post_type->name . '-titles-metas' ) . '">';
+
+		$toggle_icon = 'dashicons-arrow-up-alt2';
+		$class 		 = 'toggleable-container';
+
+		if ( $id !== 'post' ) {
+			$toggle_icon = 'dashicons-arrow-down-alt2';
+			$class .= ' toggleable-container-hidden';
+		}
+
+		printf(
+			'<h2 id="%s">%s (<code>%s</code>) <button class="toggleable-container-trigger"><span class="toggleable-container-icon dashicons %s"></span></button></h2>',
+			esc_attr( $post_type->name ),
+			esc_html( $plural_label ),
+			esc_html( $post_type->name ),
+			$toggle_icon
+		);
+
+		echo '<div class="' . $class . '">';
+
+		// translators: %s is the singular version of the post type's name.
+		echo '<h3>' . esc_html( sprintf( __( 'Settings for single %s URLs', 'wordpress-seo' ), $single_label ) ) . '</h3>';
+
 		$view_utils->show_post_type_settings( $post_type );
-		echo '</div>';
+
+		if ( $post_type->has_archive === true ) {
```

Comment:
I think this line is the cause of #10211. When using `==` it works.

Reply:
Just created a PR #10212 to fix that. The problem here is that the `has_archive` property can be a string or a boolean but the code matches for the type as well.